backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -12,13 +12,17 @@ Because all taken backup remains until expired, also check the time to live (TTL
 
 
 You can back up applications by creating a `Backup` custom resource (CR). For more information, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR].
+The following are the different backup types for a `Backup` CR:
 
 ifdef::openshift-rosa,openshift-rosa-hcp[]
 The `Backup` CR creates backup files for Kubernetes resources and internal images on S3 object storage.
 endif::openshift-rosa,openshift-rosa-hcp[]
 
 ifndef::openshift-rosa,openshift-rosa-hcp[]
 * The `Backup` CR creates backup files for Kubernetes resources and internal images on S3 object storage.
+* In an on-cluster backup, you back up the applications and persistent data within the cluster. This type of backup is fast, and you can execute it frequently; however, the data is vulnerable to cluster or storage outages. You can use Container Storage Interface (CSI) volume snapshots for an on-cluster backup.
+* In an off-cluster backup, you back up the Openshift object data and persistent data and move them from a cluster to ensure that a cluster outage does not impact the backup. This type of backup is slower than an on-cluster backup; and therefore you should execute it less frequently than an on-cluster backup. You can use CSI Data Mover backup and Kopia File System Backup (FSB) for an off-cluster backup.
```

Comment:
I think it should be whether you would like to store persistent volume data in bucket or not.  Yamls are always stored in bucket irrespective of type of backup